Sources/Layout/MessageSizeCalculator.swift

```diff
@@ -97,7 +97,8 @@ open class MessageSizeCalculator: CellSizeCalculator {
         let avatarVerticalPosition = avatarPosition(for: message).vertical
         let accessoryViewHeight = accessoryViewSize(for: message).height
         let accessoryViewVerticalPadding = accessoryViewPadding(for: message).vertical
-        let accessoryViewTotalHeight = accessoryViewHeight + accessoryViewVerticalPadding
+        let accessoryViewYOriginInset = cellTopLabelHeight + messageTopLabelHeight + messageVerticalPadding + messageContainerPadding(for: message).top
+        let accessoryViewTotalHeight = accessoryViewHeight + accessoryViewVerticalPadding + accessoryViewYOriginInset
```

Comment:
Care to explain what you're doing here? I'll review later but the logic was already super confusing before this 🤔

Reply:
Adding the extra distance that the accessoryView's origin starts at

Reply:
`messageVerticalPadding` already includes `messageContainerPadding(for: message).top`. Not sure if I really understand this approach? Shouldn't the logic be the same for resolving the height against the `AvatarView` when it is positioned `.messageCenter`? 

If you think you have a better way of calculating it -- I'm open to suggestions but I can't understand what trick you're using so I'm skeptical 🤔

Reply:
Idk if this is a better way. I was just trying to come to a solution. I can adjust